### PR TITLE
Update INSTALL.md

### PR DESCRIPTION
**Pull request by capiman2**
Update INSTALL.md to get it compiling again and make it clearer how to compile

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -9,8 +9,9 @@
 * C library development libraries and header files
 * pkg-config
 * OpenSSL >= 1.0.2
-* tpm2-tss >= 2.2.2
+* tpm2-tss >= 3.1.0-dev
```
Comment (AndreasFuchsSIT) [inline INSTALL.md:12]: This should be >= 2.4.x

**Review by AndreasFuchsSIT — CHANGES_REQUESTED** [review]
Thanks a lot for the update.
Especially Documentation is highly appreciated !

One minor change would be required: tpm2-tss >= 2.4.0

Thanks,
Andreas

**Comment by capiman2** [comment]
Changed my patch to use 2.4.x instead of 3.1.0-dev

**Review by AndreasFuchsSIT — CHANGES_REQUESTED** [review]
Sorry for bothering you again.
But I can only merge this if the change comes in a single commit (intermediate states shall be clean if possible) and if there is a signed-off-by cotribution agreement.

This is how you do this:
- Please squash both commits into one commit
```
git rebase -i HEAD~2
change the first word in the second line to squash in the opened editor
```
- Please add a signed-off-by statement
```
git commit -s --amend
```
- Then force-push your change
```
git push --force
```

Then I'll merge it right away.

**Comment by capiman2** [comment]
Created a new pull request

**Pull request closed without merging**